setup-client.yml

```diff
@@ -1,6 +1,14 @@
 - hosts: arbeitsplatz
   roles:
   - ssh-keys
+  - role: firefox-client
+    vars:
+      firefox_extensions:
```

Comment:
Ich würde die Variablen gerne in die group_vars/arbeitsplatz.yml schreiben

Reply:
Erledigt